docs/rfc/rfc-1.adoc

```diff
@@ -43,6 +43,42 @@ collects tBTC fees it uses those fees to buy T tokens back for its treasury.
 
 == In Depth
 
+[[group-size]]
+=== Group Size
+There are three main governable parameters that we need to choose:
+* The number of signers required to sign messages `threshold`
```

Comment:
Worth confirming in tss-lib and the original paper if `threshold+1` are required to sign a message and up to `threshold` can be corrupted so that the key is not in danger. Not something blocking this PR but stating it precisely could avoid confusion during the implementation.

Reply:
In v1, we set our `threshold`param to `2` so that we require 3-of-3, so I think it makes sense to follow that same definition! Updated in 682e5cb